gallery/src/components/NxToast/NxToastFullPageVariationsExample.tsx

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { useState, ReactElement } from 'react';
+import {reject, propEq} from 'ramda';
+import {
+  NxGlobalSidebarNavigation,
+  NxGlobalSidebarNavigationLink,
+  NxPageSidebar,
+  NxButton,
+  NxButtonBar,
+  NxFontAwesomeIcon,
+  NxBackButton,
+  NxStatefulGlobalSidebar,
+  NxPageMain,
+  NxPageTitle,
+  NxH1,
+  NxToastContainer,
+  NxToast,
+  NxSuccessAlert,
+  NxErrorAlert,
+  NxInfoAlert,
+  NxWarningAlert,
+  NxTile,
+  NxP
+} from '@sonatype/react-shared-components';
+import {
+  faArrowLeft,
+  faArrowRight,
+  faLink,
+  faQuestionCircle,
+  faBell,
+  faCog,
+  faUserCircle
+} from '@fortawesome/free-solid-svg-icons';
+
+const sidebarLogoPath = require('../../assets/images/logo-plaid-villain-text.png');
+
+const alerts = {
+  success: NxSuccessAlert,
+  error: NxErrorAlert,
+  info: NxInfoAlert,
+  warning: NxWarningAlert
+};
+
+type alertType = keyof typeof alerts;
+
+type ToastModel = {
+  toastId: number,
+  alert: ReactElement;
+};
+
+export default function NxToastFullPageExample() {
```

Comment:
```suggestion
export default function NxToastFullPageVariationsExample() {
```

Reply:
renamed function to NxToastFullPageVariationsExample here: [c1e07de](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/c1e07de714219ad666458562a5b9e494da737abd)